specification/billing/resource-manager/Microsoft.Billing/stable/2021-10-01/examples/PaymentMethodOwnedByUser_List.json

```diff
@@ -0,0 +1,61 @@
+{
+  "parameters": {
+    "api-version": "2021-10-01"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/providers/Microsoft.Billing/paymentMethods/ABCDABCDABC0",
+            "name": "ABCDABCDABC0",
+            "type": "Microsoft.Billing/paymentMethods",
+            "properties": {
+              "accountHolderName": "abc",
+              "expiration": "1/2035",
+              "lastFourDigits": "1270",
+              "family": "CreditCard",
+              "type": "mc",
+              "displayName": "Master Card",
+              "logos": [
+                {
+                  "mimeType": "image/png",
+                  "url": "https://contoso.com/staticresourceservice/images/v4/logo_mc_rect.png"
+                },
+                {
+                  "mimeType": "image/svg+xml",
+                  "url": "https://contoso.com/staticresourceservice/images/v4/logo_mc.svg"
+                }
```

Comment:
Who is consuming these images? Is it really needed for the API?

Reply:
Payment method blade in portal.azure.com is consuming these logos to be shown to user along with other data like last four digits, expiration, etc. URL of these images/logos are store in backend payment system.